application/src/test/resources/agent_set.yml

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2024 Telefonaktiebolaget LM Ericsson
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+connection:
+  cql:
+    agent:
+      enabled: false
```

Comment:
What will happen when no "normal" cql config is given and the agent is not anabled?

Reply:
It must use the default configs, used for sidecar mode, the code for it will be done on task #673